pkg/webui/containers/collaborator-form/index.js

```diff
@@ -61,11 +61,23 @@ const validationSchema = Yup.object().shape({
       }),
     })
     .test('collaborator is not empty', sharedMessages.validateRequired, emptyCollaboratorCheck),
+  member: Yup.object()
```

Comment:
The fact that we call it `Member` should be purely for display purposes and not change the shape of the form values or the validation at all. This should just stay `collaborator` internally.

Reply:
The reason for this is that the error message, when the field is empty, will say "collaborator is required", which is odd if the displayed name of the field is "Member"

Reply:
I see. You should be able to achieve this also by creating a new message `Member is required` that you then use instead of `validateRequired` which by default references the field name.